From ef3fbe15805325aca54c035a0fef45e6a03f8c99 Mon Sep 17 00:00:00 2001
From: Marcel Metz <mmetz@adrian-broher.net>
Date: Thu, 29 Sep 2016 13:21:29 +0200
Subject: [PATCH] Remove -Wl,--no-undefined linker flag

Checking for support gives a false positive with the AppleClang compiler
and causes a compiler error later on.
---
 CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1615ae4..aadf53a 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -390,9 +390,6 @@ if(USE_GCC OR USE_CLANG)
   set(CMAKE_REQUIRED_FLAGS "-Wl,--no-undefined")
   check_c_compiler_flag("" HAVE_NO_UNDEFINED)
   set(CMAKE_REQUIRED_FLAGS ${ORIG_CMAKE_REQUIRED_FLAGS})
-  if(HAVE_NO_UNDEFINED)
-    list(APPEND EXTRA_LDFLAGS "-Wl,--no-undefined")
-  endif()
 endif()
 
 if(ASSEMBLY)
-- 
2.7.4

